### PR TITLE
Add disclaimer for customized built-ins support

### PR DESCRIPTION
**Pull request by scottjehl**
### Description

This is for issue #30968. It adds a brief reminder to check browser support for the IS attribute when considering using this feature, as WebKit has not yet implemented the feature.

### Motivation

As it is currently written, this page quite convincingly introduces standard customized built-ins as the first of two ways to add custom element behavior, but it doesn't make it clear that only the second way ("Autonomous custom elements") has baseline browser support. 

Unfortunately, folks may end up building with this feature without knowing that Webkit has not yet implemented this part of the standard. A reminder when the feature is first introduced will keep folks from being led down a path that could lead to breakage.

### Related issues and pull requests

This is for issue #30968

### CODE REVIEW

File: files/en-us/web/api/web_components/using_custom_elements/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This article introduces custom elements, and walks through some examples.
 
 There are two types of custom element:
 
-- **Customized built-in elements** inherit from standard HTML elements such as {{domxref("HTMLImageElement")}} or {{domxref("HTMLParagraphElement")}}. Their implementation customizes the behavior of the standard element.
+- **Customized built-in elements** inherit from standard HTML elements such as {{domxref("HTMLImageElement")}} or {{domxref("HTMLParagraphElement")}}. Their implementation customizes the behavior of the standard element. _Note: Please see browser support for the complementary [`is`](/en-US/docs/Web/HTML/Global_attributes/is) attribute when considering usage._
```
Comment (pepelsbey) [inline files/en-us/web/api/web_components/using_custom_elements/index.md:17]:
```suggestion
- **Customized built-in elements** inherit from standard HTML elements such as {{domxref("HTMLImageElement")}} or {{domxref("HTMLParagraphElement")}}. Their implementation customizes the behavior of the standard element.
  > **Note:** Please see [browser support](/en-US/docs/Web/HTML/Global_attributes/is#browser_compatibility) for the complementary `is` attribute when considering usage.
```

I’d suggest using the special “Note” banner and linking directly to the compatibility table. What do you think?

Comment (scottjehl) [reply files/en-us/web/api/web_components/using_custom_elements/index.md:17]: Sure, that's even better. Thanks!

Also, should this part say that it extends the behavior for instances of the custom element instead?
`Their implementation customizes the behavior of the standard element.`

Change to:

`Their implementation extends the behavior of select instances of the standard element.`

That seemed possibly more accurate based on my understanding.

File: files/en-us/web/api/web_components/using_custom_elements/index.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ This article introduces custom elements, and walks through some examples.
 
 There are two types of custom element:
 
-- **Customized built-in elements** inherit from standard HTML elements such as {{domxref("HTMLImageElement")}} or {{domxref("HTMLParagraphElement")}}. Their implementation customizes the behavior of the standard element. _Note: Please see browser support for the complementary [`is`](/en-US/docs/Web/HTML/Global_attributes/is) attribute when considering usage._
+- **Customized built-in elements** inherit from standard HTML elements such as {{domxref("HTMLImageElement")}} or {{domxref("HTMLParagraphElement")}}. Their implementation customizes the behavior of the standard element.
```
Comment (scottjehl) [inline files/en-us/web/api/web_components/using_custom_elements/index.md:17]: Curious if we should change this to:

> Their implementation extends the behavior of select instances of the standard element.

**Review by estelle — APPROVED** [review]
Thank you for contributing to MDN/content. Welcome to the team!

File: files/en-us/web/api/web_components/using_custom_elements/index.md (hunk 2 above)
Comment (sideshowbarker) [reply files/en-us/web/api/web_components/using_custom_elements/index.md:17]: > Curious if we should change this to:
> 
> > Their implementation extends the behavior of select instances of the standard element.

That seems fine. You want to go ahead and make that change in this PR branch before we merge?

Comment (pepelsbey) [reply files/en-us/web/api/web_components/using_custom_elements/index.md:17]: I agree, it sounds better. Thanks!

Comment (scottjehl) [reply files/en-us/web/api/web_components/using_custom_elements/index.md]: Great! Done, thanks!

**Comment by scottjehl** [comment]
Thanks @estelle!

**Pull request merged**